modules/gdnative/gdnative/rect2.cpp

```diff
@@ -31,6 +31,7 @@
 #include "gdnative/rect2.h"
 
 #include "core/math/rect2.h"
+#include "core/math/rect2i.h"
```

Comment:
I didn't bother splitting things in GDNative as this code is going to be removed.